packages/devops/scripts/lambda/actions/spin_up_tupaia_database.py

```diff
@@ -26,7 +26,11 @@ def spin_up_tupaia_database(event):
     db_instance_type = event.get('DbInstanceType', 'db.t4g.large')
     security_group_code = event.get('SecurityGroupCode', 'tupaia-dev-sg') # Use security group tagged with code
     clone_db_from = event.get('CloneDbFrom', 'production') # Use volume snapshot tagged with deployment name
-    extra_tags = build_extra_tags(event)
+    extra_tags = build_extra_tags(
+        event,
+        # Turn off overnight, but come back online an hour before the server so db is available
+        { 'StopAtUTC': '09:00', 'StartAtUTC': '18:00' }
```

Comment:
This time seems appropriate 👍 

Means we're losing an hour on the current server uptime, but that's not necessarily a bad thing.

Just wanna note though, the current eventbridge event to do reclone the databases also occurs at 18:00 UTC, which means that it will clash with this time and most likely fail. Should be fine if we bump it back by 30 mins to 18:30 UTC?

Reply:
Yeah I figured actually we don't need them to come online so early - we set it earlier when the build time was taking over an hour, so it wouldn't be up by the time people started work. These days the server comes online really quickly.

Good call on bumping back the reclone!